sessionctx/variable/sysvar.go

```diff
@@ -434,37 +450,29 @@ func init() {
 	initSynonymsSysVariables()
 }
 
-// BoolToOnOff returns the string representation of a bool, i.e. "ON/OFF"
-func BoolToOnOff(b bool) string {
-	if b {
-		return BoolOn
-	}
-	return BoolOff
-}
-
-func int32ToBoolStr(i int32) string {
-	if i == 1 {
-		return BoolOn
-	}
-	return BoolOff
-}
-
-func checkCharacterValid(normalizedValue string, argName string) (string, error) {
-	if normalizedValue == "" {
-		return normalizedValue, errors.Trace(ErrWrongValueForVar.GenWithStackByArgs(argName, "NULL"))
-	}
-	cht, _, err := charset.GetCharsetInfo(normalizedValue)
-	if err != nil {
-		return normalizedValue, errors.Trace(err)
-	}
-	return cht, nil
-}
```

Comment:
These functions are not related to the struct so they were just moved to varsutil.go.